bin/install-carthage

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+version=0.9.4
+
+curl -L -O "https://github.com/Carthage/Carthage/releases/download/$version/Carthage.pkg"
```

Comment:
Hmm, good call. I haven't _seen_ that problem (using this same technique on a client app right now), but I'll keep an eye on it.